app/views/posts/index.html.erb

```diff
@@ -11,7 +11,7 @@
 
   <div class="container">
     <% @posts.each do |post| %>
-    <div class="card container mx-auto p-4 my-3">
+    <div class="card container mx-auto p-4 my-3 my_card">
```

Comment:
Prefer `-` over `_` for css classes. 

https://stackoverflow.com/questions/1547986/can-i-use-camel-case-in-css-class-names
Refer to Answers 1 and 2 in the stackoverflow question